test/plugin/test_in_udp.rb

```diff
@@ -272,7 +272,7 @@ def create_udp_socket(host, port)
       format none
       message_length_limit #{message_length_limit}
     !)
-    d.run(expect_records: 3) do
+    d.run(expect_records: 3, timeout: 10) do
```

Comment:
It might be better that we can set more small timeout value.
Even on CI environment, all tests are finished within 2 seconds.
So it seems that around `5` is enough for this.
```
2024-10-20T09:17:05.8108294Z UdpInputTest: 
2024-10-20T09:17:05.8132638Z   test:  configure w/o parse section:                      .: (0.002319)
2024-10-20T09:17:05.8160748Z   test: configure[ipv4]:                           .: (0.002736)
2024-10-20T09:17:05.8186787Z   test: configure[ipv6]:                           .: (0.002556)
2024-10-20T09:17:06.8431205Z   test: message size with format[none]:                    .: (1.024340)
2024-10-20T09:17:07.9689944Z   test: message size with format[json]:                    .: (1.125782)
2024-10-20T09:17:09.0005725Z   test: message size with format[regexp]:          .: (1.031455)
2024-10-20T09:17:19.5253499Z   test: message_length_limit:                              .: (10.524729)
2024-10-20T09:17:19.5287904Z   test: message_length_limit/body_size_limit compatibility[message_length_limit]:  .: (0.003373)
2024-10-20T09:17:19.5315054Z   test: message_length_limit/body_size_limit compatibility[body_size_limit]:       .: (0.002647)
2024-10-20T09:17:20.6871566Z   test: receive_buffer_size:                               .: (1.155556)
2024-10-20T09:17:21.8155892Z   test: remove_newline:                                    .: (1.128371)
2024-10-20T09:17:22.9269816Z   test: source_address_key:                                .: (1.111325)
2024-10-20T09:17:24.0369114Z   test: source_hostname_key:                               .: (1.109844)
2024-10-20T09:17:25.1467710Z   test: time_format[ipv4]:                         .: (1.109725)
2024-10-20T09:17:26.2572103Z   test: time_format[ipv6]:                         .: (1.110337)
```

Reply:
Agreed!

Reply:
Thanks. 
I updated timeout value to 5 sec.